.github/CODEOWNERS

```diff
@@ -68,12 +68,12 @@
 # ServiceOwner: @Azure/azure-sdk-write-keyvault
 # ServiceLabel: %KeyVault
 # PRLabel: %KeyVault
-/sdk/keyvault/           @Azure/azure-sdk-write-keyvault @heaths
+/sdk/keyvault/           @Azure/azure-sdk-write-keyvault @Azure/azure-sdk-rust-dev
```

Comment:
This is overkill. I wanted to stay on here because I have traditionally owned it and using it as a sort of test for core changes.